notion-distcopy/src/main/scala/com/ambiata/notion/distcopy/FileDistCopy.scala

```diff
@@ -0,0 +1,92 @@
+package com.ambiata.notion.distcopy
+
+import com.ambiata.mundane.control.RIO
+import com.ambiata.mundane.io._
+import com.ambiata.notion.core._
+import com.ambiata.poacher.hdfs.Hdfs
+import com.ambiata.saws.s3.{S3Address, S3Prefix, _}
+import org.apache.hadoop.fs._
+
+import scalaz.Scalaz._
+
+/**
+ * Copy functions between Hdfs and S3
+ */
+object FileDistCopy {
```

Comment:
These might need a separate discussion in terms of purpose, specifically there is not gain here from using dist-copy - it won't actually be distributed because that is done based on files, so not sure if it is worth having (I know some things upstream do things like this from the command line - but I think that is more a lack of CLI tools than technical benefit).